linker_scripts/code_script.txt

```diff
@@ -1041,9 +1041,13 @@ SECTIONS
     ovl_En_Niw : AT(RomLocation)
     {
         build/src/overlays/actors/ovl_En_Niw/z_en_niw.o(.text)
-        build/asm/overlays/ovl_En_Niw_data.o(.data)
+        build/src/overlays/actors/ovl_En_Niw/z_en_niw.o(.data)
+#ifdef NON_MATCHING
```

Comment:
You have a NON_EQUIVALENT in your actor, so you shouldn't be using a NON_MATCHING here